scripts/cy-wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+if test -z "$BASH_VERSION"; then
+  echo "Please run this script using bash, not sh or any other shell." >&2
+  exit 1
+fi
+
```

Comment:
Shouldn't we add a pre-check for required commands like `jq`, `wget`, `sed`, etc. ?

Reply:
I will put a message on top of the script to say this is expected to be used in our docker image. You can still use it but it's not intended to